geoutils/pointcloud.py

```diff
@@ -0,0 +1,77 @@
+"""Module for point cloud manipulation."""
+
+import warnings
+from typing import Literal
+
+import geopandas as gpd
+import numpy as np
+from scipy.interpolate import griddata
+
+from geoutils._typing import NDArrayNum
+
+
+def _grid_pointcloud(
+    pc: gpd.GeoDataFrame,
+    grid_coords: tuple[NDArrayNum, NDArrayNum],
+    data_column_name: str = "b1",
```

Comment:
Why is the default "b1"? Should it be None instead?

Reply:
It's the same default of "band1" as for the `to_pointcloud()` function and `PointCloud()` class, but it's true it doesn't need a name here (like when creating a `gpd.GeoDataFrame`), so I guess if the default could be "None" and choose the first band by default...
Need to think a bit more on it... We can still adjust this during the `PointCloud` PR + release :wink: